lib/operations/connect.js

```diff
@@ -490,7 +490,11 @@ function createTopology(mongoClient, topologyType, options, callback) {
       return;
     }
     try {
-      AutoEncrypter = require('mongodb-client-encryption')(require('../../index')).AutoEncrypter;
+      let mongodbClientEncryption = require('mongodb-client-encryption');
+      if (typeof mongodbClientEncryption.extension === 'function') {
```

Comment:
Note: I was considering instead of doing this type-safety, just attempting to do:

```js
const extension = require('mongodb-client-encryption').extension;
if (typeof extension !== 'function') {
  throw new MongoError('loaded version of `mongodb-client-encryption` does not have property `extension`. Please make sure you are loading the correct version of `mongodb-client-encryption`');
}
```

or something like that.